pkg/store/tsdb_test.go

```diff
@@ -347,6 +348,9 @@ func TestTSDBStore_LabelValues(t *testing.T) {
 			end: func() int64 {
 				return timestamp.FromTime(maxTime)
 			},
+			Matchers: []storepb.LabelMatcher{
+				{Type: storepb.LabelMatcher_EQ, Name: "foo", Value: "test"},
+			},
```

Comment:
I suggest removing this matcher as it doesn't help anything. We can get the same result without matchers.